Source/Scene/Globe.js

```diff
@@ -465,6 +465,30 @@ define([
             tileProvider.oceanNormalMap = this._oceanNormalMap;
             tileProvider.enableLighting = this.enableLighting;
 
+            surface.initialize(frameState);
+        }
+    };
+
+    /**
+     * @private
+     */
+    Globe.prototype.update = function(frameState) {
+        if (!this.show) {
+            return;
+        }
+
+        var context = frameState.context;
+        var width = context.drawingBufferWidth;
+        var height = context.drawingBufferHeight;
+
+        if (width === 0 || height === 0) {
+            return;
+        }
```

Comment:
>...but is this a check we should be doing scene-wide?

The code in `Globe` is some of the oldest in Cesium and this was probably a workaround that was never removed. I'm going to remove it since, you're right, this should be a scene-wide check.